### DISCUSSION TITLE
what setting to put blocks on automatic?

### DISCUSSION THREAD

**Discussion by Samuelllopes12** [maruohon/litematica#655] [Q&A]
what is the configuration that makes putting the blocks in automatic with just pressing the right mouse button, with blocks in the inventory?

**Comment by maruohon** [comment]
The Easy Place mode, with the "hold" option enabled allows you to just hold right click and look at the schematic blocks to place them. But you will need to individually look at every block. Also note that you **must** initially click on a schematic block for the easy place function to activate properly.

If you want more automatic than that in survival, then also get one of the separate printer extension mods:
* https://github.com/aria1th/litematica-printer
* https://github.com/aleksilassila/litematica-printer

I don't know really anything of how these mods work, and what the differences between these two are.
